ush/config.sh.RRFS_dev2_retro

```diff
@@ -0,0 +1,84 @@
+MACHINE="jet"
+ACCOUNT="nrtrr"
+EXPT_BASEDIR="/mnt/lfs4/BMC/rtwbl/mhu/rrfs/dev2"
+EXPT_SUBDIR="RRFS_dev2_retro"
+
+QUEUE_DEFAULT="batch"
+QUEUE_HPSS="service"
+QUEUE_FCST="batch"
+
+VERBOSE="TRUE"
+
+RUN_ENVIR="nco"
+PREEXISTING_DIR_METHOD="rename"
+
+EMC_GRID_NAME="GSD_HRRR3km"  # For now (20200130), this is maps to PREDEF_GRID_NAME="EMC_CONUS_coarse".
+GRID_GEN_METHOD="ESGgrid"
+
+QUILTING="TRUE"
+USE_CCPP="TRUE"
+CCPP_PHYS_SUITE="FV3_GSD_SAR"
+FCST_LEN_HRS="12"
+LBC_SPEC_INTVL_HRS="1"
+DA_CYCLE_INTERV="1"
+RESTART_INTERVAL="1 2 3 4 5 6"
+
+HALO_BLEND=10
+
+WTIME_RUN_FCST="02:45:00"
+WTIME_RUN_ANAL="00:25:00"
+
+DATE_FIRST_CYCL="20200904"
+DATE_LAST_CYCL="20200909"
+CYCL_HRS=( "00" "12" )
+
+DO_DACYCLE="TRUE"
+DO_RETRO="TRUE"
+OBSPATH="/mnt/lfs4/BMC/wrfruc/Ruifang.Li/data/obs_rap"
+OBSPATH_NSSLMOSIAC="/mnt/lfs4/BMC/wrfruc/Ruifang.Li/data/reflectivity"
+LIGHTNING_ROOT="/mnt/lfs4/BMC/wrfruc/aback/lightning_for_rrfs/fake_vaisala"
+ENKF_FCST="/mnt/lfs4/BMC/wrfruc/Ruifang.Li/data/enkf/atm_v15"
+
+EXTRN_MDL_NAME_ICS="HRRRX"
+EXTRN_MDL_NAME_LBCS="RAPX"
+
+envir="para"
+
+NET="RRFS_CONUS"
+
+ARCHIVEDIR="/5year/BMC/wrfruc/rrfs_dev2"
+NCARG_ROOT="/apps/ncl/6.5.0-CentOS6.10_64bit_nodap_gnu447"
+NCL_HOME="/home/rtrr/RRFS/graphics"
+NCL_REGION="conus"
+MODEL="RRFS_dev2"
+
+#
+# In NCO mode, the following don't need to be explicitly set to "FALSE" 
+# in this configuration file because the experiment generation script
+# will do this (along with printing out an informational message).
+#
+#RUN_TASK_MAKE_GRID="FALSE"
+#RUN_TASK_MAKE_OROG="FALSE"
+#RUN_TASK_MAKE_SFC_CLIMO="FALSE"
+
+RUN="RRFS_dev2"
+COMINgfs=""
+
+STMP="/mnt/lfs4/BMC/rtwbl/mhu/rrfs/dev2/stmp"  # Path to directory STMP that mostly contains input files.
+PTMP="/mnt/lfs4/BMC/rtwbl/mhu/rrfs/dev2/ptmp"  # Path to directory STMP that mostly contains input files.
```

Comment:
These should be updated to real-time areas.

Reply:
This for retro experiment and should point to personal space who runs retro.